tests/integration/test_integration.py

```diff
@@ -199,12 +201,16 @@ async def test_given_tls_requirer_is_deployed_with_3_units_and_related_when_prov
             timeout=1000,
         )
 
-        action_output = await run_get_certificate_action(ops_test)
+        get_certificate_action_output = await run_get_certificate_action(ops_test)
```

Comment:
run_get_certificate_action method returns {'return-code': 0} if the action is successful.

Reply:
I'm not sure I get this comment :)

Reply:
NP. I was trying to indicate test failure.